tests/TraceContext/W3CTestService/symfony-setup

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Install Symfony
+curl -sS https://get.symfony.com/cli/installer | bash
+mv /root/.symfony/bin/symfony /usr/local/bin/symfony
+
+# Set up Symfony test application
+cd tests/TraceContext/W3CTestService
+rm -rf test_app
+composer create-project symfony/skeleton test_app
+
+# Move our files into the test application
+cp index.php test_app/public/index.php
+cp TestController.php test_app/src/Controller/TestController.php
+cd test_app
+
+composer require annotations
+composer require open-telemetry/opentelemetry
+
+# Start the server to start listening in the background
+symfony server:start -d --port=8001
+
+# Setup the TraceContext Test Service
+apt-get install -y python3-pip 
+pip3 install aiohttp
+
+cd ..
+rm -rf trace-context
+git clone https://github.com/w3c/trace-context.git
```

Comment:
Trying to encapsulate what we talked about in the spec meeting, this is where you could add the symbolic link to your local repo for testing.  You'd accomplish this with 3 steps:

1)  Add a docker-compose-w3c-test.yaml to the root of this repo.  Note lines 7 and 8 in this [example](https://github.com/open-telemetry/opentelemetry-php/blob/main/docker-compose.prometheus.yaml#L7-L8) - you'd essentially be attaching a local volume to some spot on this docker container for testing `/tmp/ot-php/` or something like that)

2)  Add that yaml file to your Make function, similar to [this](https://github.com/open-telemetry/opentelemetry-php/blob/main/Makefile#L27)

3)  add a symlink at this point to that created directory:
    `ln -s /foo/bar/baz/w3c/vendor/opentelemetry-php /tmp/ot-php/`


That should have your local directory on your machine be mounted as a volume to the w3c testing service `/vendor/opentelemetry-php` folder.

Happy to work through this with you if you get stuck!